lib/board.rb

```diff
@@ -0,0 +1,54 @@
+class Board
+  require 'pry'
+
+  attr_accessor :cells
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def initialize
+    reset! 
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} " 
+  end
+
+  def position(input)
+    cells[input.to_i-1]
+  end
+
+  def position_taken?(input)
+    if cells[input.to_i-1] == " " 
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression. This should be all you need to do 

```ruby 
def position_taken?(input)
  cells[input.to_i - 1] == " "
end
```